### PR TITLE
BB-462 Expiring non-versioned objects fails

### PR DESCRIPTION
**Pull request by nicolas2bert**
**Note1: The change won't be forward-ported into Artesca branches.**
Note2: Only buckets with versioning enabled can be configured for replication.

To expire non-versioned objects correctly, we currently rely on the InvalidBucketState error returned when the S3C Backbeat API is used for a non-versioned bucket's object, until CLDSRV-461 is implemented to add support for non-versioned buckets to the Backbeat API.

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbeat",
-  "version": "7.10.13",
+  "version": "7.10.14",
```
Comment (jonathan-gramain) [inline package.json:3]: Can you do the version bump in a separate commit

Comment (nicolas2bert) [reply package.json:3]: Done.

**Comment by nicolas2bert** [comment]
@bert-e reset

**Review by jonathan-gramain — APPROVED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Comment by nicolas2bert** [comment]
@bert-e approve
